app/controllers/kudos_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class KudosController < ApplicationController
+  before_action :set_kudos, only: %i[show edit update destroy]
+  before_action :authenticate_employee!, except: %i[index show]
```

Comment:
this is not matching Acceptance Criteria:
_The root route is kudos#index accessible only to authenticated users_

Reply:
I corrected it